giskard/ml_worker/websocket/listener.py

```diff
@@ -221,10 +224,15 @@ async def wrapped_callback(req: ActionPayload, client_params: Dict, worker_info,
     return websocket_actor_callback
 
 
+@websocket_actor(MLWorkerAction.abort, execute_in_pool=False)
+def on_abort(ml_worker: MLWorkerInfo, params: AbortParams, *args, **kwargs):
+    POOL.cancel(params.job_id)
```

Comment:
What if the job does not exist

Reply:
There's a check inside `WorkerPoolExecutor.kill_task`:
```
if future is not None and not future.cancel():
```
giskard/utils/worker_pool.py:272


so nothing will  be done in case the job doesn't exist, no error either